csrc/preseg_passes/remove_empty.cpp

```diff
@@ -104,7 +104,7 @@ class EmptyTensorRemover : public DeadCodeRemover {
           tv->toString());
       auto shape = noReductionShape(tv);
       auto dtype = tv->getDataType().value();
-      auto new_tv = zeros(shape, dtype);
+      auto new_tv = zeros(shape, dtype, /*maybe_symbolic=*/false);
```

Comment:
Without the `maybe_symbolic=false` arguments, this pass would re-introduce symbolic IDs by calling factory methods `full` and `zeros`. That caused numerous test failures, such as in `StandaloneEye`. This argument lets us default to using `Iteration` for symbolic axes in those factory ops.